### DISCUSSION TITLE
[@types/cytoscape-context-menus] Inactive Reviewer for #63471

### DISCUSSION THREAD

**Discussion by jonatanschneider** [DefinitelyTyped/DefinitelyTyped#65992] [Issues with a @types package] [answered]
Hi,
the following PR has been opened and fixed for a long time: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63741
Sadly, the reviewer @navya9singh  requested a change and from there on did not interact in any way with the PR.

The requested change is not neccesary, as explained in the PR-discussion, but due to the inactivity of @navya9singh we can not merge this.

[a comment by an automated account is not shown]

**Reply by navya9singh** [reply, marked as answer]
I apologize for the delay. Just approved it right now.
